src/main/resources/hudson/plugins/git/GitSCM/buildEnv.properties

```diff
@@ -3,6 +3,7 @@ GIT_PREVIOUS_COMMIT.blurb=The hash of the commit last built on this branch, if a
 GIT_PREVIOUS_SUCCESSFUL_COMMIT.blurb=The hash of the commit last successfully built on this branch, if any.
 GIT_BRANCH.blurb=The remote branch name, if any.
 GIT_LOCAL_BRANCH.blurb=The local branch name being checked out, if applicable.
+GIT_CHECKOUT_DIR.blurb=The directory that the repository will be checked out to.
```

Comment:
Could you note in this description that the value is only set if the relative target directory extension is used?  I'd rather not have user complaints that they expected GIT_CHECKOUT_DIR to always be set.

Should it also note that it contains the value assigned to the relative target directory, which is most often a relative path from the workspace directory.

Reply:
I added a comment to say the env var will contain the value in Checkout to a sub-directory. The help for this already talks about it being a relative path, so I would think that part is covered...I hope